books/book.js

```diff
@@ -37,3 +37,42 @@ function displayBook(book) {
 
   main.innerHTML = currentBook;
 }
+
+var date = new Date();
+var dd = String(today.getDate()).padStart(2, '0');
+var mm = String(today.getMonth() + 1).padStart(2, '0');        //January is 0!
+var yyyy = today.getFullYear();
+
+var Price = document.getElementById("Price").value;
+
+function purchaseBook() {
+  const purchase = {
+    total_price: [Price],      //get element id
+    date: mm + '/' + dd + '/' + yyyy,     //actual ,today date
```

Comment:
Please check, but I think you need a "-" character instead of "/" for the date format